common/hooks/post-install/14-fix-permissions.sh

```diff
@@ -0,0 +1,30 @@
+# This hook fixes permissions in common places
+
+change_file_perms() {
+	local dir="${PKGDESTDIR}${1}"
+	# permission mask for matching the files
+	local permmask="$2"
+	# permissions which will be set on matched files
+	local perms="$3"
+	if [ -d "$dir" ]; then
+		find "$dir" -type f -perm "/$permmask" -exec chmod -v "$perms" {} +
+	fi
+}
+
+hook() {
+	[ -n "$nofixperms" ] && return 0
```

Comment:
So `nofixperms` also implies not checking perms at all, correct? I wonder if we should have `nocheckperms` for this step, and then `nofixperms` below?

Reply:
I'm not sure this level of granularity is actually necessary, just a suggestion.

Reply:
good idea, I modified it a bit to make it a bit more intuitive in my opinion:

* `nocheckperms` If set, xbps-src will not fail on common permission errors (world writable files, etc.)
* `nofixperms` If set, xbps-src will not fix common permission errors (executable manpages, etc.)

So the variables are entirely independent now, if nocheckperms is set, only fix will run, if both are set, the hook will not do anything, etc.